timers.js

```diff
@@ -1,4 +1,54 @@
+var timer = {
+  start:  document.getElementById('start'),
+  reset:  document.getElementById('reset'),
+  stop:   document.getElementById('start'),
+  pause:  document.getElementById('pause'),
+  timerElement:  document.getElementById('timer'),
+  seconds: 0,
+  timerID: 0,
 
+  changeName: function(){
```

Comment:
I might rename these methods to be a little more descriptive, e.g. `changeButtonToStart` / `changeButtonToStop`